apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.tsx

```diff
@@ -1,89 +1,91 @@
-        label: '',
-        description: '',
-        type: UpdateQuestionRequest.type.TEXT,
-        subgroup: '',
-        uniqueId: '',
-        uniqueName: '',
-        defaultLabelInReport: '',
-        defaultRdbTableName: '',
-        rdbColumnName: '',
-        includedInMessage: false,
-        messageVariableId: '',
-        labelInMessage: '',
-        codeSystem: '',
-        hl7DataType: ',',
-        requiredInMessage: false,
-        dataMartColumnName: '',
-        adminComments: '',
-        codeSet: 'PHIN',
-        relatedUnits: 'No',
-        futureDates: 'No',
-        displayType: '',
-        reportLabel: '',
-        defaultValue: 'test@gmai.com',
-        fieldLength: '50',
-        mask: 'TXT',
-        fieldSize: '50',
-        tooltip: '',
-        displayControl: 0,
-        minValue: 0,
-        maxValue: 0,
-        allowFutureDates: false,
-        dateFormat: ''
-    };
-    const validation = {
-        label: { required: true, error: false, fb: false },
-        description: { required: false, error: false, fb: false },
-        subgroup: { required: true, error: false, fb: false },
-        uniqueId: { required: true, error: false, fb: false },
-        uniqueName: { required: true, error: false, fb: false },
-        defaultLabelInReport: { required: true, error: false, fb: false },
-        defaultRdbTableName: { required: true, error: false, fb: false },
-        rdbColumnName: { required: true, error: false, fb: false },
-        dataMartColumnName: { required: false, error: false, fb: false },
-        messageVariableId: { required: true, error: false, fb: false },
-        labelInMessage: { required: true, error: false, fb: false },
-        codeSystem: { required: true, error: false, fb: false },
-        hl7DataType: { required: true, error: false, fb: false },
-        tooltip: { required: true, error: false, fb: false },
-        displayType: { required: true, error: false, fb: false },
-        fieldLength: { required: true, error: false, fb: false },
-        minValue: { required: true, error: false, fb: false },
-        maxValue: { required: true, error: false, fb: false },
-        dateFormat: { required: true, error: false, fb: false }
-    };
+namespace QuestionRequest {
+    export enum codeSet {
+        LOCAL = 'LOCAL',
+        PHIN = 'PHIN'
+    }
+}
 
+const init = {
+    label: '',
+    description: '',
+    minValue: 0,
+    maxValue: 50,
+    displayControl: 0,
+    relatedUnitsLiteral: 'ML',
+    relatedUnitsValueSet: 2,
+    includedInMessage: false
+};
+
+export type QuestionFormType = {
+    defaultLabelInReport?: string;
+    datamartColName?: string;
+    unitType?: string;
+    dateFormat?: number;
+    displayType?: string;
+    includedInMessage?: boolean;
+    requiredInMessage?: boolean;
+    messageVariableId?: string;
+    messageLabel?: string;
+    hl7DataType?: string;
+    HL7Segment?: string;
+    relatedUnits?: string;
+    allowFutureDates?: string;
+};
```

Comment:
Issue: When I attempted to create a `Numeric` question, an error was returned by the API due to a missing report label.
Upon inspecting the request that was sent: 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/88d92144-e170-4d7a-9087-781f5ad2bc08)

I saw a `defaultLabelInReport` is being sent with a value of `Test`. This does not match the user input but also does not properly map to the field the API is expecting. 

The proper fields are present in generated `ReportingInfo.ts`.